packages/webchat/src/main.tsx

```diff
@@ -97,13 +96,9 @@ class Web extends React.Component<MainProps> {
     this.config = this.extractConfig()
     this.props.updateConfig!(this.config)
 
-    if (this.config.exposeStore) {
-      const storePath = this.config.chatId ? `${this.config.chatId}.webchat_store` : 'webchat_store'
-      set(window.parent, storePath, this.props.store)
-    }
-
+    // is this necessary ?
     if (this.config.containerWidth) {
```

Comment:
I think some use it instead of adding custom CSS. We could deprecate it at some point tho!